invenio_app_ils/demo_data/items.json

```diff
@@ -0,0 +1,75 @@
+[
+  {
+    "pid": "1",
+    "shelf": "530.145.26 DEL",
+    "medium": "paper",
+    "status": "CAN_CIRCULATE",
+    "barcode": "CM-B00065507",
+    "legacy_id": "2706047",
+    "document_pid": "d1",
+    "internal_location_pid": "ilocid-1",
+    "legacy_library_id": "6",
+    "circulation_restriction": "FOUR_WEEKS"
+  },
+  {
+    "pid": "2",
+    "shelf": "004.421.5 DEL",
+    "medium": "paper",
+    "status": "CAN_CIRCULATE",
+    "barcode": "CM-B00065300",
+    "legacy_id": "2705205",
+    "document_pid": "d2",
+    "internal_location_pid": "ilocid-1",
+    "legacy_library_id": "6",
+    "circulation_restriction": "FOUR_WEEKS"
+  },
+  {
+    "pid": "3",
+    "shelf": "530.12 EIN",
+    "medium": "paper",
```

Comment:
invalid medium, please check all of them to be one of: https://github.com/inveniosoftware/invenio-app-ils/blob/master/invenio_app_ils/records/api.py#L174